<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>eVBL</class>
 <widget class="QWidget" name="eVBL">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1347</width>
    <height>916</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>eVBL</string>
  </property>
  <widget class="QComboBox" name="device_list">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>331</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="capture_frame">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>370</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Capture</string>
   </property>
  </widget>
  <widget class="CVImageWidget" name="previewVideo" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>320</width>
     <height>240</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="zoom_setting">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>40</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
   <property name="currentText">
    <string>25%</string>
   </property>
   <item>
    <property name="text">
     <string>25%</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>50%</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>75%</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>100%</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>150%</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>200%</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Fit</string>
    </property>
   </item>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>70</y>
     <width>964</width>
     <height>724</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <widget class="CVImageWidget" name="display_capture_frame" native="true">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>2</y>
      <width>960</width>
      <height>720</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CVImageWidget</class>
   <extends>QWidget</extends>
   <header>cvimagewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
